common/beerocks/bwl/dwpal/ap_wlan_hal_dwpal.cpp

```diff
@@ -2368,10 +2368,10 @@ bool ap_wlan_hal_dwpal::process_dwpal_event(char *buffer, int bufLen, const std:
         FieldsToParse fieldsToParse[] = {
             {NULL /*opCode*/, &numOfValidArgs[0], DWPAL_STR_PARAM, NULL, 0},
             {NULL, &numOfValidArgs[1], DWPAL_STR_PARAM, NULL, 0},
-            {(void *)&msg->params.success, &numOfValidArgs[2], DWPAL_CHAR_PARAM, "success=", 0},
+            {(void *)&msg->params.success, &numOfValidArgs[2], DWPAL_INT_PARAM, "cac_status=", 0},
```

Comment:
I haven't investigated how dwpal_string_to_struct_parse() is implemented, but I assume the parameter type is used to determine the size of the argument pointer. So if it is changed to int, the type of msg->params.success has to be changed to int (or unsigned) as well. That said, I don't see what was wrong with char.